.github/workflows/bundles-kit.yml

```diff
@@ -7,7 +7,7 @@ on:
         required: true
         type: string
       java-version:
-        default: '21'
+        default: '21.0.1'
```

Comment:
Don't you need to update as well the build, early-access and release workflows?

Reply:
see: `error: release version 21.0.1 not supported` in the GitHub actions
Also `build.yml` uses jdk 21?